### PR TITLE
First Draft Baha'i Website

### PR DESCRIPTION
**Pull request by sxh275**
Initial pages for index, intro, calendar, espanol and links added.  CSS and images initial pass added.  First upload of website draft to Main.

### CODE REVIEW

File: calendar.html [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+                    </ul>
+                  </div>
+                </div>
+                <div class="rd-navbar-collapse"><a class="btn btn-sm btn-dark" href="contact.html">Contact Us ➝</a></div>
+              </div>
+            </div>
+          </nav>
+        </div>
+      </header>
+
```
Comment (dhemmat) [inline calendar.html]: Let's remove unneeded enters

File: css/custom.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/* index */
+
+.brand-logo-dark {
+	margin-right:15px;
```
Comment (dhemmat) [inline css/custom.css]: replace tabs with two spaces

**Review by dhemmat — CHANGES_REQUESTED** [review]
Shidan, can we fix the two issues? After that it looks ready to merge.

File: css/custom.css (hunk 2 above)
Comment (sxh275) [reply css/custom.css]: done

**Review by dhemmat — APPROVED** [review]
This is ready to merge

**Pull request merged**